lib_xua/src/core/ports/audioports.xc

```diff
@@ -136,3 +136,67 @@ void ConfigAudioPorts(
 #endif //#if (I2S_CHANS_DAC != 0) || (I2S_CHANS_ADC != 0)
 }
 
+
+void DeConfigAudioPorts(
+#if (I2S_CHANS_DAC != 0) || (DSD_CHANS_DAC != 0)
+    out_buffered_port_32_t p_i2s_dac[],
+    int numPortsDac,
+#endif
+#if (I2S_CHANS_ADC != 0)
+    in_buffered_port_32_t p_i2s_adc[],
+    int numPortsAdc,
+#endif
+#if (CODEC_MASTER == 0)
+    NULLABLE_RESOURCE(out_buffered_port_32_t, p_lrclk),
+    out_buffered_port_32_t p_bclk,
+#else
+    NULLABLE_RESOURCE(in_buffered_port_32_t, p_lrclk),
+    in_buffered_port_32_t p_bclk,
+#endif
+    NULLABLE_RESOURCE(in_port_t,  p_mclk_in),
+    clock clk_audio_bclk)
+{
+#if (I2S_CHANS_DAC != 0) || (I2S_CHANS_ADC != 0)
+    int tmp; 
+    /* We always expect a mclk pin when xcore is master */
+    assert(!isnull(p_mclk_in));
+
+    /* Note handling of MCLK is done outside of XUA by user */
+
+    /* Clear modes/reset port and set to Hi-Z */
+    stop_port(p_bclk);
```

Comment:
not an important point, but you can just do start_port() to reset it

Reply:
Ah yes, I had that in mind but wanted belt and braces.